### ISSUE TITLE
Global modules not installed correctly

### ISSUE THREAD

**Issue by adamreisnz** [tj/n#546]
# Issue Template:

### Describe issue including what OS you are using
Raspbian Stretch Lite (latest, clean install)

Installing global npm modules doesn't seem to work correctly anymore. The modules are installed in `/usr/local/n/versions/node/10.15.0/lib/node_modules` and don't show up in `/usr/lib/node_modules`, which should be fine to keep global modules separate, however the modules do not get linked correctly and can't be run.

Have tried reinstalling of `npm` as per the readme, and also tried rebooting, to no avail.

### Describe what version of N you have
2.1.12

### Describe how you installed N
`npm install -g n`

### Steps to reproduce issue _1.2.3_
1. n 10.15.0
2. npm i -g pm2
3. pm2

### Describe the results you received
`-bash: pm2: command not found`

### Describe the results you expected
For pm2 to run.

### What version of npm you are using
6.4.1

### Additional information you deem important (e.g. issue happens only occasionally):
* Output of `which n:` /usr/bin/n
* Output of `node -v:` v10.15.0
* Output of `npm bin -g`:

```
/usr/local/n/versions/node/10.15.0/bin
(not in PATH env variable)
```

**Comment by adamreisnz** [comment]
Reinstalled everything from scratch with Node 8 and didn't run into this issue.

**Comment by JohnRGee** [comment]
What you are seeing is not the intended behaviour. 

The default install by `n` is to `/usr/local`:
* `npm` and `node` are installed in `/usr/local/bin`
* global modules go in `/usr/local/lib/node_modules`

This does not require changing `PATH` if /`usr/local/bin` is included by default.

Since you are seeing mention of `/usr/local/n/versions/node/10.15.0` this suggest somehow you are running node or npm by default out of that folder. The `n use` command does allow running a single command using the cached downloaded version, but does not directly affect other commands (like `npm bin -g`).

**Comment by JohnRGee** [comment]
If you don't have ideas about what happened with the setup, suggested first things to check are:

    which npm
    which node

**Comment by adamreisnz** [comment]
I assumed at first it had to do with node 10, but maybe it was just a fluke. Regardless, I spent a few hours trying to fix the issues but it just got worse and worse so I decided to start over instead.

I reinstalled everything following exactly the same steps as for all our ARM devices (as per a guide I compiled a while ago). Second time around I installed node 8 first, and then node 10. Everything worked ok.

Will close for now, but will keep an eye on it when I do the next install.
